smart-doc gradle plugin3.0.1 generate document failed

## Your Environment(您的使用环境)

<!--- Include as many relevant details about the environment you experienced the bug in -->

* smart-doc version:
* plugin version (e.g. smart-doc-maven-plugin or smart-doc-gradle-plugin): 3.0.1
* build tool version(maven or gradle): gradle
* jdk version:11

## Expected Behavior(您期望的结果)

<!--- If you're describing a bug, tell us what should happen -->
使用gradle插件输出postman文档，遇到两个问题
1、提示ERROR: The codePath can't be empty.
从源码看codePath必需要配置，但default.json没有给出样例，参考测试代码配置了一个避免了这个报错，但不确定配置的是否正确
2、报如下错，class文件路径多了一个L，不知道是什么原因
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':app:smartDocPostman'.
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)
Caused by: java.lang.NoClassDefFoundError: Lcom/xxxx/xxxx/xx/xxx/xxx/xxxx;
	at com.ly.doc.utils.JavaClassUtil.getFieldGenericType(JavaClassUtil.java:802)
<!--- If you're suggesting a change/improvement, tell us how it should work -->
为了使用解析swagger tag的特性从2.7.0升级到了3.0.1，在2.7.7下面是可以正常生成文档的
## Current Behavior(当前结果)

<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->

## Possible Solution(bug解决建议)

<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->

## Steps to Reproduce (Bug产生步骤，请尽量提供用例代码)

<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
除了outPath外，配置项中增加了下面两个配置项
  "codePath": "/src/main/java",
  "sourceCodePaths": [
    {
      "path": "src/main/java"
    }
  ]

## Context(Bug影响描述)

<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->

--- a/src/main/java/com/ly/doc/utils/JavaClassUtil.java
+++ b/src/main/java/com/ly/doc/utils/JavaClassUtil.java
@@ -817,7 +817,7 @@ public class JavaClassUtil {
             f.setAccessible(true);
             Type t = f.getGenericType();
             return StringUtil.trim(t.getTypeName());
-        } catch (NoSuchFieldException | ClassNotFoundException e) {
+        } catch (NoSuchFieldException | ClassNotFoundException | NoClassDefFoundError e) {
             return null;
         }
     }

